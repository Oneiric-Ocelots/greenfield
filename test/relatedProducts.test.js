import React from "react";
import { mount, shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
// import "babel-polyfill";
configure({ adapter: new Adapter() });

import AddToMyOutfitCardButton from "../src/components/RelatedItemsComponents/AddOutfitCardButton.jsx";

let sampletTest = (a, b) => {
  return a + b;
};

const Test = () => {
  return (
    <div className="testtest">
      <p>hello</p>
    </div>
  );
};

//adddd commentss
let testingtesting = "hello";

test("adds 1+2 to equal to 3", () => {
  expect(sampletTest(1, 2)).toBe(3);
});

test("AddToMyOutfitCardButton test", () => {
  expect(
    shallow(<Test />).contains(
      <div className="testtest">
        <p>hello</p>
      </div>
    )
  ).toBe(true);
});
