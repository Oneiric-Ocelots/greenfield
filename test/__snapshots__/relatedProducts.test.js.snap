// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test that MyOutfitsListlooks like the latest commmit Displays 1`] = `
<div
  style="padding:0 60px;height:375px;width:800px;margin:0 auto"
>
  <p
    class="MuiTypography-root MuiTypography-body1"
  >
    MY OUTFITS
  </p>
  <div
    class="sc-bdVaJa imYUvI"
  >
    <div
      class="sc-bwzfXH gwZiig"
    >
      <div
        class="sc-htpNat grBOFc"
        style="transform:translateX(0px)"
      >
        <div
          class="sc-bxivhb gbgaCl"
          width="-60"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation1 MuiCard-root item-card-box makeStyles-cardSpecial-1 MuiPaper-rounded"
            id="addOutfit-card-box"
          >
            <button
              class="MuiButtonBase-root MuiCardActionArea-root item-card-box-action-area"
              tabindex="0"
              type="button"
            >
              <div
                class="MuiCardContent-root makeStyles-content-3"
                id="addOutfit-card-content"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-icon-4 MuiSvgIcon-fontSizeLarge"
                  color="black"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-icon-4 MuiSvgIcon-fontSizeLarge"
                  color="black"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
                <h4
                  class="MuiTypography-root MuiTypography-h4"
                >
                   ADD TO MY OUTFITS
                </h4>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-icon-4 MuiSvgIcon-fontSizeLarge"
                  color="black"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-icon-4 MuiSvgIcon-fontSizeLarge"
                  color="black"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                  />
                </svg>
              </div>
              <span
                class="MuiCardActionArea-focusHighlight"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`test that PopOut like the latest commmit Matches the snapshot 1`] = `
<div
  class="PopOut"
>
  <button
    aria-label="popOut"
    class="MuiButtonBase-root MuiIconButton-root"
    tabindex="0"
    type="button"
    variant="contained"
  >
    <span
      class="MuiIconButton-label"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root"
        color="blanchedalmond"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
        />
      </svg>
    </span>
  </button>
</div>
`;

exports[`test that itemlist looks like the latest commmit Displays 1`] = `
<div
  style={
    Object {
      "height": 375,
      "margin": "0 auto",
      "padding": "0 60px",
      "width": 800,
    }
  }
>
  <WithStyles(ForwardRef(Typography))>
    RELATED ITEMS
  </WithStyles(ForwardRef(Typography))>
  <r
    activeItemIndex={0}
    activePosition="center"
    alwaysShowChevrons={false}
    calculateActualTranslateX={[Function]}
    chevronWidth={70}
    className="item-list-carousel"
    classes={Object {}}
    disableSwipe={false}
    enablePlaceholder={false}
    firstAndLastGutter={true}
    gutter={30}
    infiniteLoop={false}
    leftChevron={<Memo(NavigateBeforeIcon) />}
    numberOfCards={3}
    numberOfPlaceholderItems={0}
    onActiveStateChange={null}
    outsideChevron={true}
    placeholderItem={null}
    requestToChangeActive={[Function]}
    rightChevron={<Memo(NavigateNextIcon) />}
    showSlither={true}
    slidesToScroll={1}
  />
</div>
`;
